This change requires a minor version increase for package `org.apache.jackrabbit.oak.commons`.

The build failed for `oak-commons` with:
```
[ERROR] org.apache.jackrabbit.oak.commons: Version increase required; detected 2.1.0, suggested 2.2.0
```

Please update `package-info.java` in package `org.apache.jackrabbit.oak.commons`.

--- a/oak-commons/src/main/java/org/apache/jackrabbit/oak/commons/package-info.java
+++ b/oak-commons/src/main/java/org/apache/jackrabbit/oak/commons/package-info.java
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@Version("2.1.0")
+@Version("2.2.0")
 package org.apache.jackrabbit.oak.commons;
 
 import org.osgi.annotation.versioning.Version;

